### DISCUSSION TITLE
need some help for WDRVI calc

### DISCUSSION THREAD

**Discussion by veonline** [sertit/eoreader#192] [Q&A] [answered]
I'm getting an exception when trying to calculate the WDRVI index from a S2 product, it throws 'cause missing "alpha" parameter that i have no idea how to pass from the product's load method.
scavenging the code I've seen that when it calls spyndex.computeIndex it does not pass the **kwargs down the call chain, but i'm really weak on python so i don't know if it's a bug or it's my lack of experience with the language and there is a way to specify the "alpha" it asks for the index calculation.
any help really appreciated

**Comment by remi-braun** [comment, marked as answer]
This is a bug, I can reproduce it ✔️
This will be fixed in the next release, see https://github.com/sertit/eoreader/issues/193 for the corresponding issue.
Thanks a lot for reporting this!

**Comment by remi-braun** [comment]
This should be solved, you can install the github version if needed.

Basically, you should do:

```python
prod.load("WDRVI", alpha=1)
```
